ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -704,6 +704,27 @@ tests:
     test:
     - ref: fips-check-optional-operator-images-scan
     workflow: cucushift-installer-rehearse-aws-ipi-proxy
+- as: aws-ipi-proxy-sdn-workers-rhel8-fips-f14
+  cron: 21 17 10,25 * *
```

Comment:
suggest to lower the frequency to f14.

Reply:
Hmm, can we keep 14?  because I do not add more sdn profile this time, only this one on aws

Reply:
we also have a destructive beside this one, if it is on f28, that means at least 2 jobs in one month, of course, if you think that is not enough, on your call.

Reply:
@zhaozhanqi The test name has fips but there are no env to enable fips.

Reply:
@zhaozhanqi The test name has fips but there are no env to enable fips.

Reply:
this is a fips cluster with rhel worker node, so `FIPS_ENABLED` is set in the install workflow, but not set it in the job config level.

Reply:
Ack

Reply:
> we also have a destructive beside this one, if it is on f28, that means at least 2 jobs in one month, of course, if you think that is not enough, on your call.

yes, hmm, (2 normal and 1 destructive) per 28 days. should be ok I think.

Reply:
cucushift check `.data` in `configmap/cluster-config-v1`( in namespace `kube-system`) to determine whether or not to include fips test cases. So it should be fine.